src/locales/en.json

```diff
@@ -1,117 +1,116 @@
-      "time": "Time",
-      "more": "More"
-    },
-    "footer": {
-      "nervos_foundation": "Nervos Foundation",
-      "about_us": "About Us",
-      "faq": "FAQ",
-      "developer": "Developer",
-      "api": "API",
-      "gitHub": "GitHub",
-      "whitepaper": "Whitepaper",
-      "community": "Community",
-      "twitter": "Twitter",
-      "blog": "Blog",
-      "telegram": "Telegram",
-      "reddit": "Reddit",
-      "youtube": "YouTube",
-      "forum": "Forum"
-    },
-    "search": {
-      "empty_result": "Opps! Your search did not match any record. Please try different keywords~"
-    },
-    "address": {
-      "address": "Address",
-      "lock_hash": "Lock Hash",
-      "lock_script": "Lock Script",
-      "balance": "Balance",
-      "pending_reward": "Pending Reward",
-      "code_hash": "Code Hash",
-      "args": "Args",
-      "confirmation": "Confirmation",
-      "confirmations": "Confirmations",
-      "unable_decode_address": "Unable to decode address",
-      "pending_reward_tooltip": "The block reward and transaction fee of this block will be sent to the miner after 11 blocks，learn more from our Consensus Protocol"
-    },
-    "transaction": {
-      "transaction": "Transaction",
-      "transactions": "Transactions",
-      "transaction_fee": "Transaction Fee",
-      "status": "Status",
-      "input": "Input",
-      "output": "Output",
-      "lock_script": "Lock Script",
-      "type_script": "Type Script",
-      "data": "Data",
-      "base_reward": "Base Reward",
-      "secondary_reward": "Secondary Reward",
-      "commit_reward": "Commit Reward",
-      "proposal_reward": "Proposal Reward",
-      "cellbase_help_tooltip": "The cellbase transaction of block N is sent to the miner of block N-11 as reward, learn more from our Consensus Protocol",
-      "capacity": "Capacity",
-      "detail": "Detail"
-    },
-    "block": {
-      "block": "Block",
-      "block_height": "Block Height",
-      "proposal_transactions": "Proposal Transactions",
-      "block_reward": "Block Reward",
-      "uncle_count": "Uncle Count",
-      "epoch": "Epoch",
-      "epoch_start_number": "Epoch Start Number",
-      "epoch_length": "Epoch Length",
-      "nonce": "Nonce",
-      "proof": "PoW Proof",
-      "timestamp": "Timestamp",
-      "miner": "Miner",
-      "difficulty": "Difficulty",
-      "transactions_root": "Transactions Root",
-      "witnesses_root": "Witnesses Root",
-      "pending_tip": "The block reward of this block will be sent to the miner after 11 blocks，learn more from our Consensus Protocol",
-      "calculating_tip": "The transaction fee of this block will be sent to the miner after 11 blocks，learn more from our Consensus Protocol"
-    },
-    "error": {
-      "maintain": "The site is temporarily down for maintenance. Please check back soon."
-    },
-    "pagination": {
-      "first": "First",
-      "last": "Last",
-      "goto": "Goto",
-      "page": "Page"
-    }
-  }
+	"translation": {
```

Comment:
Why add more indent here?

Reply:
Use spaces instead of tabs.

Reply:
hmm, not sure how this got into my commit. I didn't touch the file